docs/Ways-of-Working/Toolkit/RAID.md

```diff
@@ -2,82 +2,143 @@
 title: RAID
 tags: 
  - draft
- - Challenge Mapping
+ - RAID
  - Default
 ---
 
 
-## Objective
+## Introduction
 
-Proactively manage and mitigate risks, assumptions, issues and dependencies throughout the problem lifecycle.
+- RAID capture, management and mitigation is a continuous process throughout the entire problem lifecycle. 
+- If you are unsure of how to approach RAID capture, management and mitigation, the following guide documents a default approach.
+- This may be a useful starting point for delivery teams, and as competency grows, teams are encouraged to experiment with, define and evolve their own processes and procedures whilst operating within the stipulated minimum viable guardrails.
+- The guidance demonstrates several options for managing RAID so that teams can find what works best for them. If a team is unsure where to start they are encouraged to arrange a short session with a RAID SME who, through discussion, can make recommendations on the best approach for the team.
 
-## RAID
+## RAID SME's
```

Comment:
More out of my curiosity, do we have the term SME defined anywhere?
I don't know what it means but I'm guessing from the context "Subject Matter Expert"?